js/constants/constants.js

```diff
@@ -155,6 +155,23 @@ module.exports = {
     MULTI_AUDIO_OPTIONS: 'Closed Multi Audio Options'
   },
 
+  CONTROL_BAR_KEYS: {
+    PLAY_PAUSE: 'playPause',
+    LIVE: 'live',
+    VOLUME: 'volume',
+    TIME_DURATION: 'timeDuration',
+    FLEXIBLE_SPACE: 'flexibleSpace',
+    MORE_OPTIONS: 'moreOptions',
+    QUALITY: 'quality',
+    DISCOVERY: 'discovery',
+    CLOSED_CAPTION: 'closedCaption',
```

Comment:
Should we rename this to closedCaptions to make it consistent with other closed captions constants? Though I already notice a few discrepancies like discover vs discovery

Reply:
Yes, I think `closedCaptions` is the correct name, but this matches the name from the buttons section of the skin.json. I also kept the value that was already being used in case it was being referenced elsewhere.

Reply:
@aeng7 actually, it looks like the multi audio and VR360 ones don't follow this rule, I'll fix them. But the idea is that they need to match the name from the skin.json